lang_go/parsing/parser_go.mly

```diff
@@ -758,6 +758,16 @@ complitexpr:
 |   "{" braced_keyval_list "}" { InitBraces ($1, $2, $3) }
 
 bare_complitexpr:
+(* Special case to deal with automatic semicolon insertion. We want to parse
```

Comment:
I usually add a sgrep: or semgrep: tag in the comment, just to indicate that this is parsing code used only
for semgrep.
There is also a Flag_parsing.sgrep_guard that can enforce things.